physionet-django/user/test_views.py

```diff
@@ -15,7 +15,7 @@
 from django.contrib.messages.storage.fallback import FallbackStorage
 from django.core import mail
 from django.core.management import call_command
-from django.core.files.uploadedfile import SimpleUploadedFile
+from django.core.files.uploadedfile import SimpleUploadedFile, UploadedFile
```

Comment:
This line is changed but the change doesn't seem necessary. I don't see UploadedFile being used?

Reply:
Ah yes, that should not be there, let me make changes.